### ISSUE TITLE
The Window is not responding

### ISSUE THREAD

**Issue by georg-lam** [microsoft/vscode#165903]
**The problem**
When I start Visual Studio Code I only get a window stating "The window is not responding. You can reopen or close the window or keep waiting".

**My analysis**
The only problem that I detected in the log output (_code . --verbose_) is 

_CodeWindow: detected unresponsive_

**Version Information**
I installed Visual Studio Code 1.73.0 for the first time on "Microsoft Windows Server 2016 Standard" (Version 10.0.14393 Build 14393). 

**Current workaround**
When I start VSCode with --no-sandbox then it seems to work normally. 

**Question**
How can I further troubleshoot this issue so that eventually I can run VScode with sandboxing.

**Comment by bpasero** [comment]
Are you maybe running VS Code as an admin user?

**Comment by georg-lam** [comment]
Yes, this is correct. The users working on this server are admins.
